### PR TITLE
Create cloud-load-balancer-troubleshooting.md

### PR DESCRIPTION
**Pull request by rsgeinzer**
This is to provide our customers basic trouble shooting for cloud load balancers.  This is being done per the request of the CLB Guild process.  Purpose is to reduce time for basic troubleshooting and knowledge.

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-fb15bbf362/how-to/). :bow:

**Comment by catlook** [comment]
jira ticket: ID 1005.  Should pick it up in our next sprint!!  Thanks :)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-1abc10f689/how-to/)

**Comment by cyrichardson** [comment]
Some suggestions here: 
[Cloud-Load-Balancers-troubleshooting_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/1742246/Cloud-Load-Balancers-troubleshooting_cyrEdits.docx)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-2c38426853/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-aa2c26605e/how-to/)

File: content/cloud-load-balancers/cloud-load-balancer-troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+last_modified_by: Cat Lookabaugh
+product: Cloud Load Balancers
+product_url: cloud-load-balancers
+---
+
+This article provides troubleshooting instructions for Cloud Load Balancers.
+
+### Use Cloud Load Balancers health monitoring
+
+To help with the trouble shooting process,  you should enable the health monitoring for Cloud Load Balancers.
```
Comment (stephamon) [inline content/cloud-load-balancers/cloud-load-balancer-troubleshooting.md]: troubleshooting (one word) and there is extra space following the comma.

File: content/cloud-load-balancers/cloud-load-balancer-troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+This article provides troubleshooting instructions for Cloud Load Balancers.
+
+### Use Cloud Load Balancers health monitoring
+
+To help with the trouble shooting process,  you should enable the health monitoring for Cloud Load Balancers. 
+This feature provides the ability to review historical load balancer actions.  If health monitoring is not enabled, few troubleshooting options exist.
+
+#### Enable Cloud Load Balancers health monitoring
+
+1. Open the customer portal for the identified load balancer.
```
Comment (stephamon) [inline content/cloud-load-balancers/cloud-load-balancer-troubleshooting.md]: Open with something like "To enable health monitoring for your load balancer, use the following steps." so we don't go from header straight to a numbered list.

File: content/cloud-load-balancers/cloud-load-balancer-troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+2. Choose **optional features**, and click the pencil icon to the right of the health monitoring line.  This opens the **Health Monitoring Settings** panel.
+3. Fill in the appropriate settings, and click **Save Monitoring Settings**.
+
+### Use Cloud Monitoring
+
+You might also want to install the Cloud Monitoring service, which is provided at no cost to monitor the health of the nodes (or virtual instances) that are attached to Cloud Load Balancers. For more information, see [Install and configure the Rackspace Monitoring Agent](https://support.rackspace.com/how-to/install-and-configure-the-rackspace-monitoring-agent/) and [Rackspace Monitoring checks and alarms](https://support.rackspace.com/how-to/rackspace-monitoring-checks-and-alarms/).
```
Comment (stephamon) [inline content/cloud-load-balancers/cloud-load-balancer-troubleshooting.md]: These links don't need to have "https://support.rackspace.com" in them. You can start with "/how-to/"

File: content/cloud-load-balancers/cloud-load-balancer-troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+### Keep Cloud Load Balancers resources in the same Datacenter
+
+For the most part, Cloud Load Balancers communicate to attached nodes by using the service net internat protocaol (IP) address.  Normally, this address is identified as starting with 10.X.X.X.  This means that the Cloud Load Balancers and any associated nodes must be located in the same data center. (This includes testing with a known operational server.)
+
+### Use Cloud Load Balancers with RackConnect
+
+RackConnect v2 and v3 are compatible with Cloud Load Balancers but require special setup considerations. For more information, see [Use Cloud Load Balancers with RackConnect](https://support.rackspace.com/how-to/using-cloud-load-balancers-with-rackconnect).
```
Comment (stephamon) [inline content/cloud-load-balancers/cloud-load-balancer-troubleshooting.md]: Same here about the link.

File: content/cloud-load-balancers/cloud-load-balancer-troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+If a load balancer suddenly stops working, it might be due to changes on the node(s) attached to the load balancer.  Review recent changes to the web service, firewall rules, and so on.  If changes were made, roll them back to see if the load balancer begins working again.  If a new node was added to the load balancer and the load balancer is automatically disabling this node, compare how the nodes were setup to identify any differences.
+     
+     
+### Use Pitchfork to view statistics
+ 
+ Pitchfork is a valuable tool that can be used to view different statistics concerning the Cloud Load Balancers in an account.
+ 
+ To learn more, see [Pitchfork - Rackspace Cloud API Web Application](https://community.rackspace.com/products/f/public-cloud-forum/6432/pitchfork---rackspace-cloud-api-web-application).
+ 
+### Cloud Load Balancers issues
```
Comment (stephamon) [inline content/cloud-load-balancers/cloud-load-balancer-troubleshooting.md]: We should probably have an introductory sentence here so we don't have a header followed by a header. Maybe something like, "Following are some common load balancer issues and solutions." (I don't love that, but you get the idea.)

File: content/cloud-load-balancers/cloud-load-balancer-troubleshooting.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+This situation normally results when the load balancer is unable to connect to the attached nodes.
+
+To resolve this condition, perform the following steps for each affected node:
+
+1. Check the status of the node behind the load balancer.
+
+2. Log into the customer portal, and open the emergency console for the node.
+     
+   If there are messages scrolling on the screen (for example, killing processes or a crash dump), you should rebooted the node.  Once the node is operational, check the status to make sure it is enabled.
```
Comment (stephamon) [inline content/cloud-load-balancers/cloud-load-balancer-troubleshooting.md]: ... you should reboot the node.

File: content/cloud-load-balancers/cloud-load-balancer-troubleshooting.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+  
+### Using cURL commands
+
+One troubleshooting method for a failing node is to log on to a good node in the same load balancer and send cURL commands to the node that is having issues.
+
+#### cURL commands
+  
+To use the following helpful commands, install cURL, and execute the commands from a terminal window.
+  
+       Test the load balancer:  ``curl - <load balancer public IP address>``
```
Comment (stephamon) [inline content/cloud-load-balancers/cloud-load-balancer-troubleshooting.md]: These should probably be in a bullet list.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-1a602dfd31/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-5b7089a7fa/how-to/). :bow:

**Comment by stephamon** [comment]
👍

**Pull request merged**